src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ArrayConverter.cs

```diff
@@ -33,7 +32,81 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
 
             return base.ConvertTo(context, culture, value, destinationType);
         }
-    }
-}
 
+#if !NETSTANDARD10
```

Comment:
Once we remove the non-generic collections (tracking by #8359) we can remove all the if/defs for targeting. I'd rather not do any more work here since it will be thrown away when we can consolidate back on .NET Standard 1.0.